pkg/model/model.go

```diff
@@ -76,6 +81,32 @@ func (m *Model) crdNames() []string {
 	return crdConfigs
 }
 
+func CheckIfFieldExists(
+	field map[string]*Field,
+	currValue string,
+) bool {
+	_, ok := field[currValue]
+	if ok {
+		return true
+	}
+	return false
+}
```

Comment:
Mmmmm i think this is unecessary.. it will always fall down to:
```go
                  if _, ok := fieldsInParent[parent]; ok {
						// Check if the field is of type structure
						previous, res = CheckType(value, fieldsInParent)
						if res != true {
							fmt.Printf("Cannot add new field here, %s is not of type Structure", parentName)
							break
						}
					}
```